helm/provisioner/templates/provisioner.yaml

```diff
@@ -146,6 +146,12 @@ apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: {{ $val.name }}
+  {{- if kindIs "map" $val.storageClass }}
+  {{- if $val.storageClass.isDefaultClass }}
```

Comment:
Thanks for the PR! Can you add a test for this under https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/tree/master/helm/examples